sdk/typescript/test/e2e/read-events.test.ts

```diff
@@ -12,25 +12,30 @@ describe('Event Reading API', () => {
   });
 
   it('Get All Events', async () => {
-    const allEvents = await toolbox.provider.getEvents("All", null, null);
+    const allEvents = await toolbox.provider.getEvents('All', null, null);
     expect(allEvents.data.length).to.greaterThan(0);
-    expect(allEvents.nextCursor).toEqual(null);
```

Comment:
removed this because this assumption no longer stays true for devnet and testnet